### PR TITLE
Log out number of files to send.

### PR DESCRIPTION
**Pull request by DanielChicot**

### CODE REVIEW

File: src/main/kotlin/app/batch/BufferingProcessor.kt [hunk 1]
```diff
@@ -11,7 +10,7 @@ import java.io.ByteArrayOutputStream
 
 @Component
 @Qualifier("buffer")
-class BufferingProcessor() : ItemProcessor<EncryptedStream, EncryptedStream> {
```
Comment (dan-hill2802) [inline src/main/kotlin/app/batch/BufferingProcessor.kt:14]: why are the parenthesis removed?

Comment (DanielChicot) [reply src/main/kotlin/app/batch/BufferingProcessor.kt:14]: It defines an empty primary constructor which is redundant - intellij code clean-up which I run before pushing suggests getting rid of them and other things like unused imports etc.

Comment (steveburton4) [reply src/main/kotlin/app/batch/BufferingProcessor.kt:14]: They are not needed in declaring the class, syntax should be to remove them so I think this is ok.

File: src/main/kotlin/app/batch/S3DirectoryReader.kt [hunk 2]
```diff
@@ -62,6 +62,10 @@ class S3DirectoryReader(private val s3Client: AmazonS3, private val s3Utils: S3U
                 request.continuationToken = results.nextContinuationToken
             } while (results != null && results.isTruncated)
 
+            logger.info("s3 object count",
```
Comment (steveburton4) [inline src/main/kotlin/app/batch/S3DirectoryReader.kt]: Can we start with a capital please? Sorry...

File: src/main/kotlin/app/batch/S3DirectoryReader.kt [hunk 3]
```diff
@@ -62,6 +62,10 @@ class S3DirectoryReader(private val s3Client: AmazonS3, private val s3Utils: S3U
...
+                    "bucket_name" to bucketName,
+                    "s3_prefix_folder" to s3Utils.s3PrefixFolder, "object_count" to objectSummaries.count().toString())
```
Comment (steveburton4) [inline src/main/kotlin/app/batch/S3DirectoryReader.kt:67]: Do we have topic name here or is it added to every single log message anyway? Would be good to log it here if not.

**Review by steveburton4 — CHANGES_REQUESTED** [review]

File: src/main/kotlin/app/batch/S3DirectoryReader.kt (hunk 3 above)
Comment (DanielChicot) [reply src/main/kotlin/app/batch/S3DirectoryReader.kt:67]: It's in every message.

File: src/main/kotlin/app/batch/S3DirectoryReader.kt (hunk 2 above)
Comment (DanielChicot) [reply src/main/kotlin/app/batch/S3DirectoryReader.kt]: done.

**Review by steveburton4 — APPROVED** [review]

**Pull request merged**